app/assets/javascripts/discourse/app/components/user-stream.js

```diff
@@ -71,6 +73,28 @@ export default Component.extend(LoadMore, {
     this._lastDecoratedElement = lastElement;
   },
 
+  _scrollToLastPosition() {
+    let scrollTo = this.session.userStreamScrollPosition;
+    if (scrollTo && scrollTo >= 0) {
+      schedule("afterRender", () => {
+        discourseDebounce(
```

Comment:
what is the point of debounce with 0  value?

Reply:
Also note this debounce is using an anonymous function so it won't have any effect

Reply:
I do not think it makes sense. It does not seem to do anything, so I just removed it. Maybe it used to fix something in the past.